From b4e7e32c3ad66aa88356e653f916dd5bcd987a1e Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Thu, 24 Aug 2023 11:21:12 +0200
Subject: [PATCH] target/linux/mvebu: Disable extra errata

In newer kernel, new errata are available and explicit decision is
needed to compile kernel. We are not affected by those, so let's disable
them.
---
 target/linux/mvebu/config-6.6 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/linux/mvebu/config-6.6 b/target/linux/mvebu/config-6.6
index a4572d8a89..3555609099 100644
--- a/target/linux/mvebu/config-6.6
+++ b/target/linux/mvebu/config-6.6
@@ -446,3 +446,5 @@ CONFIG_ZLIB_INFLATE=y
 CONFIG_ZSTD_COMMON=y
 CONFIG_ZSTD_COMPRESS=y
 CONFIG_ZSTD_DECOMPRESS=y
+CONFIG_ARM64_ERRATUM_2054223=n
+CONFIG_ARM64_ERRATUM_2067961=n
-- 
2.47.0

